packages/ui/src/components/Textarea/Textarea.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react';
+
+import { HStack } from '../Stack';
+import { Textarea, TextareaProps } from './index';
+
+const meta: Meta = {
+  title: 'Components / Textarea',
+  component: Textarea,
+};
+
+export default meta;
+
+const Template: Story<TextareaProps> = (args) => <Textarea {...args} />;
+
+export const Playground = Template.bind({});
+
+Playground.args = {
+  placeholder: 'Default textarea',
```

Comment:
It would be great to provide values for the size, disabled, value, name, and invalid. This way Storybook will initially display correct controls for them.